plugin/codeium.vim

```diff
@@ -19,7 +19,9 @@ function! s:SetStyle() abort
 endfunction
 
 function! s:MapTab() abort
-  imap <script><silent><nowait><expr> <Tab> codeium#Accept()
+  if !get(g:, 'codeium_no_map_tab', v:true)
```

Comment:
I think the default we want here is `v:false` so that `<Tab>` *is* mapped by default

Reply:
Hey, nice catch! I was having trouble finding documentation for the `get` function, and misunderstood the purpose of the third argument. I will update the default there shortly.

Reply:
@sullivan-sean I have updated, sorry for the delay